voicer-client/src/components/voice/EditProfile.jsx

```diff
@@ -60,9 +64,19 @@ export default function EditProfile({ profileData, token }) {
             <FormControl {...emailInput} placeholder="Email" />
           </InputGroup>
           <InputGroup className="mb-3">
-            <FormControl {...passwordInput} placeholder="Password" />
+            <FormControl
+              {...passwordInput}
+              placeholder="Password"
+              type="password"
+            />
+          </InputGroup>
```

Comment:
let's get rid of password for now, that's an rc2 feature anyways

Reply:
@alphaseinor Fair enough, we can address later.